src/generators/scripts/stop.js

```diff
@@ -46,7 +48,10 @@ fi
 `
 }
 
-export function stopScriptDocker() {
+export function stopScriptDocker(config) {
+  if (config.network.txGenerate) {
+    return readFileToString(joinPath(libRootDir(), 'lib', 'stop-with-splunk.sh'))
```

Comment:
this should check for splunk, not txGenerate. Again here we can just conditionally include `docker-compose -f docker-compose-splunk.yml down` below and get rid of the separate shell script